### DISCUSSION TITLE
Inline Code Color

### DISCUSSION THREAD

**Discussion by mabourehab** [nvim-neorg/neorg#1163] [Q&A] [answered]
Hi,

I'm wondering if there is a way to change the color of the inline code written between ` `. Not necessarily syntax specific coloring, but just changing the color to something more visible. Currently it looks like this:

![image](https://github.com/nvim-neorg/neorg/assets/87973909/35de7f16-df0a-4ad3-9949-e3126f9fc0dc)

As you can see the code (starting from echo) is barely distinguishable from the regular text and I want to know if I can change it to a brighter color.

Thanks!

**Comment by max397574** [comment, marked as answer]
yes just change the highlight
`:hi neorg.markup.verbatim` iirc
see what it's linked to and change this

not sure but out of my head I'd have said it's `NonText` you need to change
or just change the highlight directly

**Comment by dimas-cyriaco** [comment]
I'm using this:

```lua
vim.cmd([[hi! link @neorg.markup.verbatim @neorg.markup.superscript]])
```

**Comment by mabourehab** [comment]
Hi @dimas-cyriaco @max397574, thanks for the awesome answers! It worked!

Just an additional question, is there a way I can change also the code text to be italic or bold for example. 

Sorry for all these basic questions, I'm very new to all of this and a bit lost in all the information :)

**Reply by dimas-cyriaco** [reply]
I think you would have to define the full highlight,  and not just link, in this case. Something like this:

```lua
vim.cmd([[hi! @neorg.markup.verbatim cterm=bold,italic gui=bold,italic guifg=#ff9e64]])
```

You can use `Telescope highlights` to see the current value of the highlights.

**Reply by dimas-cyriaco** [reply]
Other useful tip, if you are in a recent neovim version, is the `:Inspect` command. I will show you the highlight group under the cursor.

**Reply by mabourehab** [reply]
Hi! Yup, that's perfect now! Thanks, alot and the Inspect command is pretty useful as well!

**Reply by max397574** [reply]
instead of vim.cmd you can (and imo should because "code logic" as string is ugly) use
`vim.api.nvim_set_hl("@neorg.markup.verbatim", {fg="#ffab00", bg="#abff00", link="OtherHlGroup", bold = true, italic = true})`
